Sources/BeeTracking/TrackingLikelihoodModel.swift

```diff
@@ -101,7 +114,16 @@ extension TrackingLikelihoodModel : McEmModel {
       // sample from noise model on manual pose
       var proposal = datum.obb.center
       proposal.perturbWith(stddev: Vector3(0.3, 8, 4.6))
-      return (1.0, proposal)
+      return (
+        // The weight of the proposal, which equals to the unnormalized probability of the likelihood:
+        self.unnormalizedProbability(
```

Comment:
I’d prefer to use a variable for the patch

Reply:
Sure, I'll do that!